lib/wor/paginate/adapters/will_paginate_already_paginated.rb

```diff
@@ -0,0 +1,32 @@
+# Used when render_paginated is called with an already paginated ActiveModel with will_paginate
+### render_paginated DummyModel.paginate(page: 1)
+
+module Wor
+  module Paginate
+    module Adapters
+      class WillPaginateAlreadyPaginated < Wor::Paginate::Adapters::Adapter
+        attr_reader :page
+
+        def required_methods
+          # Methods will_paginate adds to ActiveRecord relations:
+          ### [:current_page, :total_entries, :total_entries=, :find_last, :current_page=,
+          ### :scoped, :total_pages, :next_page, :previous_page, :out_of_bounds?]
+          %i(previous_page out_of_bounds? total_entries= total_pages current_page=)
+        end
+
+        def paginated_content
+          @paginated_content ||= @content.limit(nil).offset(0)
```

Comment:
Okey, this is awkward, with limit(nil), we are resetting the limit from the original querie, just to be sure that they are using the pagination provided by our params. Otherwise it have crazy behaviours paginating the already paginated things